pact/ns.pact

```diff
@@ -1,8 +1,8 @@
 (define-keyset 'swap-ns-user)
 (define-keyset 'swap-ns-admin)
-(ns.write-registry 'swap (keyset-ref-guard 'swap-ns-admin) true)
+(ns.write-registry (read-msg 'ns) (keyset-ref-guard 'swap-ns-admin) true)
 (define-namespace
-  "swap"
+  (read-msg 'ns)
```

Comment:
Looks good!
`(read-msg 'ns)` would get evaluated to `kswap` based off `pact/yaml/testnet/ns.yaml`